js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navAnchors = document.querySelectorAll('a');
+let mainHeader = document.querySelector('h1');
+let headerButton = document.querySelector('button');
+let headerImg = document.getElementById('cta-img');
+let H4 = document.querySelectorAll("h4");
+let paragraphs = document.querySelectorAll("p")
+let middleImg = document.getElementById("middle-img");
+
+
+let navList = document.querySelectorAll('nav a')
+for (let i = 0; i < navList.length; i++) {
+  navList[i].textContent = siteContent['nav'][`nav-item-${i + 1}`];
+  navList[i].style.color = 'green';
+};
```

Comment:
Excellent. Love it. Work that loop, honey. Imagine if we had a _thousand_ nav anchor elements? This loop saves us a lot of time and keeps our code DRY. Also very nice use of template literals with interpolation here.